project_report/templates/all_templates.html

```diff
@@ -0,0 +1,54 @@
+
+{% macro non_human_bioinf(genome_version,bcl2fastq_version,samblaster_version,gatk_version,bcbio_version,adapter2,adapter1,bwa_version) -%}
+    <h2>Bioinformatics analysis</h2>
+    <h5>Demultiplexing is performed using bcl2fastq ({{bcl2fastq_version}}), allowing 1 mismatch when assigning reads
+    to barcodes. Adapters (Read1: {{adapter1}}, Read2: {{adapter2}}) are trimmed during the demultiplexing
+    process.</h5>
```

Comment:
`<h5>` tags are meant to be used for title, here we should use `<p>` tags for standard text.
You can add css classes to the p tag to format them the way you want 
```css
p.class_name {
    font-size: 1.1em; 
    line-height: 140%; 
    font-family: Avantgarde; 
    sans-serif; 
}
```
```html
<p class="class_name"> Text </p>
```